kernel/README.md

```diff
@@ -2,7 +2,11 @@
 
 This document explains the steps to build a compatible kernel with Kata
 Containers. To do this use build-kernel.sh, this script automates the
-process to build a kernel for Kata Containers. 
+process to build a kernel for Kata Containers.
+
+## Requirements
+
+Go-lang version 1.10+ required by the setup.sh script.
```

Comment:
Personally I'd phrase it as something like '[Go](https://golang.org/) version 1.10 or above required'. Just my preference though.

Reply:
Hi @rcsavage - thanks for raising this. However, I'd prefer we don't duplicate versions like this. Instead, please could you refer to the following which explains the golang requirement and points the user to the "versions database" file which specifies all versions of external entities used by Kata:

 -https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components